src/components/alert/bl-alert.css

```diff
@@ -0,0 +1,103 @@
+:host {
+  --bl-display: block;
+
+  display: var(--bl-display);
+}
+
+.alert {
+  --bl-padding: var(--bl-size-m);
+  --bl-main-color: var(--bl-color-primary);
+  --bl-main-bg-color: var(--bl-color-accent-primary-background);
+  --bl-description-font-size: var(--bl-font-size-m);
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: var(--bl-main-bg-color);
+  color: var(--bl-color-content-primary);
+  box-shadow: 0 0 0 1px var(--bl-main-color);
+  border-radius: var(--bl-border-radius-s);
+  padding: 0 var(--bl-padding);
+}
+
+.description {
+  font-size: var(--bl-description-font-size);
+  line-height: var(--bl-description-font-size);
+}
+
+.wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex: auto;
+}
+
+.content {
+  display: flex;
+  align-items: center;
+  padding: var(--bl-padding) 0;
+}
+
+.icon {
+  margin-right: var(--bl-size-2xs);
+  color: var(--bl-main-color);
+}
+
+.text-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.caption {
+  color: var(--bl-main-color);
+  font: var(--bl-font-title-3-medium);
+  margin-bottom: var(--bl-size-4xs);
+}
+
+.actions {
+  display: flex;
+  align-items: center;
+}
+
+.caption + .description {
+  --bl-description-font-size: var(--bl-font-size-s);
```

Comment:
Use `--bl-font-caption-text` for the description font when a caption is provided. Otherwise use `--bl-font-title-3-regular`